api/server/structs/conversions.go

```diff
@@ -433,6 +433,15 @@ func (a *AttestationElectra) ToConsensus() (*eth.AttestationElectra, error) {
 	}, nil
 }
 
+func SingleAttFromConsensus(a *eth.SingleAttestation) *SingleAttestation {
```

Comment:
not a comment here but something i forgot why we did it, why do we not check if a.Data is not nil?  do we do that somewhere else? i forgot.